### PR TITLE
Feature/a11y navigation

### PR DESCRIPTION
**Pull request by afewyards**

### CODE REVIEW

**Comment by Ex-iT** [comment]
The weekdays and change month buttons are overlapping in IE10

ie10-month-buttons

File: paper-calendar-a11y-behavior.html [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+            break;
+          }
+          targetDate.setMonth(month - 1);
+          break;
+        case 'Enter':
+        case 'Space':
+          this.date = targetDate;
+          break;
+
+        case 'Escape':
```
Comment (Ex-iT) [inline paper-calendar-a11y-behavior.html]: It looks to me that pressing the escape key doesn't do anything, should it close the calendar maybe?

**Comment by Ex-iT** [comment]
Its possible to 'select' non existing days with a mouse click
non-exisiting-days

File: paper-calendar-track-behavior.html [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+      // only drag during pan-x gesture
+      if (this._gesture !== 'pan-x') {
+        return;
+      }
+
+      this._dragging = true;
+      var fullWidth = width * this._months.length;
+      var x = this._startPos + dx;
+
+      // If we're dragging outside the bounds, add some resistence
```
Comment (Ex-iT) [inline paper-calendar-track-behavior.html]: typo: `resistance`

**Comment by Ex-iT** [comment]
Swiping to change month doesn't seem to work in Firefox (v50.1.0).

File: test/paper-date-picker.html [hunk 3]
```diff
@@ -1,336 +1,322 @@
...
-            // much transformations
-            expect(calendar._currentPos).to.equal(-(calendar.$.months.clientWidth / 2));
-            done();
-          }, 500);
+      NodeList.prototype.forEach = Array.prototype.forEach;
+
+      describe('paper-date-picker', function () {
+        var element;
+
+        context('with a max-date', function () {
```
Comment (Ex-iT) [inline test/paper-date-picker.html]: Test fails in IE10 and Edge
ie-test

Comment (afewyards) [inline test/paper-date-picker.html]: was failing before as well... needs to be addressed in an other branch

**Pull request merged**